components/rabbitmq/scripts/create.py

```diff
@@ -113,7 +113,8 @@ def _install_rabbitmq():
 
     utils.systemd.start('cloudify-rabbitmq')
 
-    sleep(30)
+    time.sleep(10)
+    utils.wait_for_port(5672)
 
```

Comment:
it's a comfort more than anything to prevent us from retrying the operation just because rabbit isn't ready.